Video Player/app.js

```diff
@@ -0,0 +1,56 @@
+const player=document.querySelector(".player");
+const video=player.querySelector(".viewer");
+const progress=player.querySelector(".progress");
+const progressBar=player.querySelector(".progress__filled");
+const toggle = player.querySelector(".toggle");
+const skipButtons=player.querySelectorAll("[data-skip]");
+const ranges=player.querySelector("range");
+//player-slider: video kaydırıcı
+
+//lets write the functions
+function togglePlay(){
+    //.paused -> video is play or pause check
+    //if video paused then set ►, or ❚ ❚
+    if(video.paused){
+        video.play();
+    }
+    else{
+        video.pause();
+    } 
+}
+
+function updateButton(){
+    const icon= this.paused ? '►' : '❚ ❚';
+    console.log(icon);
+    toggle.textContent=icon;
+}
+
+//"currentTime" geçerli oynatma süresini saniye cinsinden belirtir
+function skip(){
+    video.currentTime +=parseFloat(this.dataset.skip);
+}
+
+//?
+function handleRangeUpdate(){
+    video[this.name]=this.value;
+}
+
+function handleProgress(){
+    const percent=(video.currentTime/video.duration)*100;
+    progressBar.style.flexBasis=`${percent}`;
+}
+
+// eventlistener'ları ekleyelim
+video.addEventListener('click', togglePlay);
```

Comment:
addEventListener larin hepsini bir fonksiyonda topla. Uygulama basinda o fonksiyonu cagir. 
Api de kullandigimiz mantigi uygula buna. O sekilde yazmayi tüm uygulamalarinda standart haline getir. 
Normal bir uygulama da öyle calisiyor. Uygulama icin gerekli bilesenleri yüklüyorsun ve mount fonksiyonu calisiyor. Isimlendirmeler farkli olsa da ayni mantikla calisiyor. 
Vue js createApp fonksiyonu: https://vuejs.org/guide/essentials/application.html#the-root-component
Root render react: https://www.w3schools.com/react/react_getstarted.asp
Angular : https://github.com/gothinkster/angular-realworld-example-app/blob/main/src/main.ts